docs/source/guide/tutorials/base/how_to_train/action_detection.rst

```diff
@@ -0,0 +1,144 @@
+Action Detection  model
+================================
+
+This live example shows how to easily train, validate, optimize and export spatio-temporal action detection model on the subset of `JHMDB <http://jhmdb.is.tue.mpg.de/>`_.  
+
+.. note::
+
+  To learn deeper how to manage training process of the model including additional parameters and its modification, refer to :doc:`./detection`.
```

Comment:
please, add also links to deploy and demo

Reply:
Currently demo and deploy do not support video tasks because demo.py in exportable_codes folder do not consider multi-frame inputs to model. They can get video or stream, but they offer only one frame to nn model. I'm gonna extend the demo.py to video inputs in next release. In this case should we need to inform deploy and demo section in this video part?

Reply:
@GalyaZalesskaya could you add this information to demo tutorial?